test/jdk/java/awt/Robot/HiDPIScreenCapture/HiDPIRobotScreenCaptureTest.java

```diff
@@ -115,38 +105,26 @@ public void paint(Graphics g) {
             throw new RuntimeException("Wrong image size!");
         }
 
-        checkRectColor(image, new Rectangle(0, 0, w / 2, h / 2), COLORS[0]);
-        checkRectColor(image, new Rectangle(w / 2, 0, w / 2, h / 2), COLORS[1]);
-        checkRectColor(image, new Rectangle(0, h / 2, w / 2, h / 2), COLORS[2]);
-        checkRectColor(image, new Rectangle(w / 2, h / 2, w / 2, h / 2), COLORS[3]);
+        checkRectColor(image, w / 4, h / 4, COLORS[0]);
+        checkRectColor(image, 3 * w / 4, h / 4, COLORS[1]);
+        checkRectColor(image, w / 4, 3 * h / 4, COLORS[2]);
+        checkRectColor(image, 3 * w / 4, 3 * h / 4, COLORS[3]);
```

Comment:
As my testing showed, this test was stable enough with the changes introduced by [JDK-8282270](https://bugs.openjdk.java.net/browse/JDK-8282270), #7425, with the reduced size of the frame.

Reply:
I can revert the changes for this one test, but will do that in a separate commit so it's easier to revert in case there are troubles.